### PR TITLE
Dave 21

### PR DESCRIPTION
**Pull request by crystal95**

### CODE REVIEW

File: src/main/js/electron/templates/index.html [hunk 1]
```diff
@@ -3,39 +3,45 @@
...
 <script>
-    $(document).ready(function () {
-    $( '#upload_input' ).on( 'change', function () {
-    var fullfilename=this.value;
-    var filename = fullfilename.replace(/^.*[\\\/]/, '')
-    document.getElementById("uploadFile").value = filename;
-  });
-});
-</script>
-<script>
```
Comment (pbalm) [inline src/main/js/electron/templates/index.html]: Why did you remove this section? With this removed, we don't see the filename in the "load spectrum" textbox anymore after selecting, right?

**Comment by pbalm** [comment]
Looks ok except for two things, the filename isn't showing anymore after loading (see comment above) and I think you're using the wrong column order in your input data but I'm not sure it matters. Anyway it shouldn't be hard to fix, let's see what Simone says.

**Comment by crystal95** [comment]
Ah , I added it again in DAVE-22 :) , I removed it for checking something , any ways it will be added back when we merge DAVE-22 ........... Also I will reorder the columns to suit the test file I will get :)

File: src/main/js/electron/templates/index.html (hunk 1 above)
Comment (pbalm) [inline src/main/js/electron/templates/index.html]: Added again in PR for DAVE-22
